Assets/HoloToolkit/Sharing/Tests/Scripts/ImportExportAnchorManager.cs

```diff
@@ -701,13 +693,14 @@ private void AnchorStoreReady(WorldAnchorStore store)
         /// </summary>
         private void CreateAnchorLocally()
         {
-            if (thisAnchor.isLocated)
+            WorldAnchor anchor = this.EnsureComponent<WorldAnchor>();
```

Comment:
I think I had some problem with the anchor only being crated once, in the following situation:
* Anchor is crated
* Anchor is located
* Anchor is serialized, but is rejected due to small size
* Anchor is never re-crated, so the serialization size is always the same.

If the anchor is re-crated the definition is also re-crated, and if the spatial map is now bigger the serialization will also be more accurate (and be a larger data blob). I'm not sure if re-serializing is enough when the spatial map is improved, but i got the feeling a re-creation of the anchor was necessary.